recipes/glu/all/conanfile.py

```diff
@@ -0,0 +1,65 @@
+from conans import ConanFile, tools
+from conans.errors import ConanException
+import os
+
+class SysConfigGLUConan(ConanFile):
+    name = "glu"
+    version = "system"
+    description = "cross-platform virtual conan package for the GLU support"
+    topics = ("conan", "opengl", "glu")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://cgit.freedesktop.org/mesa/glu/"
+    license = "SGI-B-2.0"
+    settings = ("os",)
+
+    def requirements(self):
+        self.requires("opengl/system")
+
+    def package_id(self):
+        self.info.header_only()
+
+    def _fill_cppinfo_from_pkgconfig(self, name):
+        pkg_config = tools.PkgConfig(name)
```

Comment:
for this to work fine, it requires that any consumer of this recipe has pkg-info tool installed. I this is something that is usually installed in the system for Linux users that's fine, but it is not the case for Windows ones

Reply:
This method is used only when building for Linux. Could it cause problems when cross-compiling from Windows to Linux (I don't have much experience with this)?

Anyway, PkgConfig code could be removed, GLU header and library have fixed locations, as described in [OpenGL ABI for Linux](https://www.khronos.org/registry/OpenGL/ABI).

Reply:
thanks for the clarification and glad we can even remove pkg-config, that will improve the compatibility for any user in Linux

Reply:
I'm not sure it's so safe to remove pkg-config usage, on ubuntu the library is not in the folder specified by khronos.org: see https://packages.ubuntu.com/eoan/amd64/libglu1-mesa-dev/filelist
On my machine, `/usr/lib/x86_64-linux-gnu/pkgconfig/glu.pc` has the following content :
```
prefix=/usr
exec_prefix=${prefix}
libdir=${prefix}/lib/x86_64-linux-gnu
includedir=${prefix}/include

Name: glu
Description: Mesa OpenGL Utility library
Requires: gl
Version: 9.0.0
Libs: -L${libdir} -lGLU
Libs.private: -lm
```
if we don't use pkg-config,we lose `-lm` too here

Reply:
Also, glu/system requires opengl/system [which uses pkg-config](https://github.com/conan-io/conan-center-index/blob/master/recipes/opengl/all/conanfile.py#L19), so pkg-config needs to be present anyway

Reply:
please keep pkg-config. various distros may install into various location, also some users tend to install into the non-standard locations (such as `/opt`, `/usr/local`, `/usr/local/opt`, etc), may install from sources, use non-standard package manager (snap, flatpak, etc.) and so on.

Reply:
Alright, I reverted the change.